### PR TITLE
[RPS-465] Add "unlocker" group

### PR DESCRIPTION
**Pull request by wojtek-oledzki**
Members of the "unlocker" group can unlock documents.

Also added 404 page for /login urls.

### CODE REVIEW

File: repository-data/local/src/main/resources/hcm-config/configuration/groups/editor.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+definitions:
+  config:
+    /hippo:configuration/hippo:groups/editor:
+      hipposys:members:
+      - unlocker
```
Comment (ziemsky) [inline repository-data/local/src/main/resources/hcm-config/configuration/groups/editor.yaml]: I'm guessing that this does _not_ mean that every member of group editor gets 'unlocker' privileges, right?

File: repository-data/application/src/main/resources/hcm-config/configuration/modules/documentworkflow.scxml [hunk 2]
```diff
@@ -115,8 +115,8 @@
                         <!-- for a not-current editor the current editor (holder) is reported through the 'inUseBy' feedback -->
                         <hippo:feedback key="inUseBy" value="holder"/>
                     </if>
-                    <if cond="!editor and workflowContext.isGranted(draft,'hippo:admin')">
```
Comment (ngta1) [inline repository-data/application/src/main/resources/hcm-config/configuration/modules/documentworkflow.scxml]: Assume removing this doesn't affect admin unlock operation?

File: repository-data/local/src/main/resources/hcm-config/configuration/users/unlocker.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+---
+definitions:
+  config:
+    /hippo:configuration/hippo:users/unlocker:
+      hipposys:active: true
+      hipposys:password: unlocker
```
Comment (ziemsky) [inline repository-data/local/src/main/resources/hcm-config/configuration/users/unlocker.yaml]: Does this mean we get a default 'unlocker' user alongside the built-in 'author' and 'editor' in vanilla installations?

What's the impact on already established installations: does such a user get added there on upgrade as well, necessitating a manual intervention to disable it in production environment?

Comment (wojtek-oledzki) [reply repository-data/local/src/main/resources/hcm-config/configuration/users/unlocker.yaml]: not vanilla - localhost only, for testing

File: repository-data/application/src/main/resources/hcm-config/configuration/modules/documentworkflow.scxml (hunk 2 above)
Comment (wojtek-oledzki) [reply repository-data/application/src/main/resources/hcm-config/configuration/modules/documentworkflow.scxml]: it does :) hence admin in "unlocker" group by default

**Pull request merged**